geometry_msgs/Pose goal
---
int64 succeed